pkgs/by-name/ri/rip2/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "rip2";
+  version = "0.9.0";
+  src = fetchFromGitHub {
+    owner = "MilesCranmer";
+    repo = pname;
```

Comment:
```suggestion
    repo = "rip2";
```

Reply:
Saw some other packages do this. Wasn’t sure what the best practice was